Improve recommendations on publication credit (#2839)
nf-core/website: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md

**Optionally**, the nf-core community can be included as a consortium co-author. This option is preferred when a pipeline has made extensive use of existing nf-core pipeline components (e.g., modules and subworkflows) written by other community members who were not directly involved in the pipeline itself. For example, see hgtseq: [A Standard Pipeline to Study Horizontal Gene Transfer](https://doi.org/10.3390/ijms232314512).

> If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation. We also recommend transparency on working on a piece where the pipeline is a major part of it. If you plan to write a paper about a pipeline, please share this with the community on the pipeline channel on the nf-core Slack workspace so that other contributors can join your effort.

Reviewer: ```suggestion
If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation. We also recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information with the community in the pipeline specific channel on the nf-core Slack workspace, so that other contributors can join and support your effort.
```

Reviewer 2: I would go even further 

```suggestion
If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.

We also strongly recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information prior and during writing of the manuscript with the community in the pipeline specific channel on the nf-core Slack workspace, so that other major contributors can join and support your effort.
```

I sort of wonder if we shiould put a caveat somewhere (cc' @ewels maybe/) that nf-core provides 'no warranty to the generation of publication' etc or something... or some wording saying while we want to facilitate openness and collabration, we are not responsible for authorship etc.

Reviewer 3: I wonder if we should show leadership here and define some criteria. We (nf-core) don't want to end up being mediators for authorship, but we could be more proactive in defining criteria.

[For example](https://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html#two):

The ICMJE recommends that authorship be based on the following 4 criteria:
- Substantial contributions to the conception or design of the work; or the acquisition, analysis, or interpretation of data for the work; AND
- Drafting the work or reviewing it critically for important intellectual content; AND
- Final approval of the version to be published; AND
- Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.

These don't really apply here, but something similar might help in being proactive rather than reactive when deciding what constitutes a contribution.

Author: I agree with all your points! I will commit Sateesh/James' contribution, but we can keep discussing how to make it even better, based on Chris' comment.

Reviewer 2: I dunno, I don't think we should be proactive from the point of:

1. in reality we aren't an entity that has any weight (we can't get lawyers involved, and yes that can happen), so can we really have any say in such things? Who is then responsible?
2. co-authorship arguments are really political and messy and we can't expect us as volunteers involved to get long drawn out arguments
3. many different co-authorship guidelines do exist (e.g. the ICMJE thing you quote), this varies per field AND is often not actually interpreted in the same way, or just out-right ignored and say 'well we've never done that before.

However I might be biased and burnt out because I've been too long in a field where these discussions  and blown up waaaaay too much and I'm sick of it :sweat_smile:

If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.

We also strongly recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information prior and during writing of the manuscript with the community in the pipeline specific channel on the nf-core Slack workspace, so that other major contributors can join and support your effort.

> If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

Author: ```suggestion
If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we developed 4 criteria:

- Substantial contributions to the conception or design of the pipeline; or the acquisition, analysis, or interpretation of data used to test/demo the pipeline; AND
- Drafting the work or reviewing it critically for important intellectual content; AND
- Final approval of the version to be published; AND
- Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.
```

Reviewer 3: I found [this link](https://prevention.ucsf.edu/sites/prevention.ucsf.edu/files/2024-07/Authorship%20Guidelines%20for%20DPS%20Community%20Engaged%20Research%20FINAL%20Doc%20071624.pdf). It expands on the criteria a little. I find the clause about not granting authorship to anyone not involved in the manuscript drafting to be a little strict. I would drop the second bullet point.

Maybe this is a discussion for the next core team meeting? Get lots of eyes on it as this is a reasonably large decision.

Reviewer 3: Didn't read but might also be worth a skim [this](https://osf.io/s6h58/download/)

Reviewer 2: ```suggestion
If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we suggest you could evaluate inclusion of contributors as a co-author based on the following 4 criteria:

    - Substantial contributions to the conception or design of the pipeline; or the acquisition, analysis, or interpretation of data used to test/demo the pipeline; AND
    - Drafting the work or reviewing it critically for important intellectual content; AND
    - Final approval of the version to be published; AND
    - Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.
```

However I still don't _really_ like this, it's being too proscriptive and I worry about might put people off contributing pipelines to nf-core if they (PIs) are worried about the risk of getting into protracted arguments about authorships or feel that it's no-longer their pipeline and they include the publication with their thesis etc.

Furthmore, it could be someone wants authorship to recognise their contribution, but they don't care about the drafting the actual publication (it's not their job), so the 'AND' is going to make that difficult. Furthermore, someone may not want to be accountable for _all_ aspects of the work if it's some other lab who has done lab work analysis etc. in the publication.

To be clear: conceptually it is nice, but I don't feel we can control this.

We already have problems tracking all developers, if we now have to chase them to make sure they are following

Reviewer 3: That's a good point. I guess it all comes back to what constitutes a substantial contribution and how to encourage thoughtful consideration for authorship in different scenarios. Maybe we could give some examples.

Something something something... substantial contribution. For example:
- Maintenance of xxxx 
- Development of xxxx
- Or other xxxx

Reviewer 2: Yeah, that would work with examples. But ensure the wording is that they are just suggestions to help you decide

Reviewer 4: @jfy133 / @mribeirodantas - shall we apply these suggested changes and then do a final review?

If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.

We also strongly recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information prior and during writing of the manuscript with the community in the pipeline specific channel on the nf-core Slack workspace, so that other major contributors can join and support your effort.

If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

> Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we suggest you could evaluate inclusion of contributors as a co-author based on the following 4 criteria:

Reviewer 3: ```suggestion
Determining fair guidelines for authorship in community-driven projects, especially those involving diverse contributors, can be challenging. Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we suggest you evaluate the inclusion of contributors as co-authors based on the following four criteria:
```

    - Substantial contributions to the conception or design of the pipeline; or the acquisition, analysis, or interpretation of data used to test/demo the pipeline; AND
    - Drafting the work or reviewing it critically for important intellectual content; AND
    - Final approval of the version to be published; AND
    - Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.

> 

Reviewer 3: ```suggestion

However, there are many different types of contributions in nf-core and these criteria may not fit every situation. If in doubt, we encourage authorship decisions to favor inclusivity and equity.

```

Reviewer 2: I would maybe put this bit in an admonition

If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

Determining fair guidelines for authorship in community-driven projects, especially those involving diverse contributors, can be challenging. Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we suggest you evaluate the inclusion of contributors as co-authors based on the following four criteria:

    - Substantial contributions to the conception or design of the pipeline; or the acquisition, analysis, or interpretation of data used to test/demo the pipeline; AND
    - Drafting the work or reviewing it critically for important intellectual content; AND
    - Final approval of the version to be published; AND
    - Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.

> However, there are many different types of contributions in nf-core and these criteria may not fit every situation. If in doubt, we encourage authorship decisions to favor inclusivity and equity.

Reviewer 4: ```suggestion
:::tip
There are many different types of contributions in nf-core and these criteria may not fit every situation. If in doubt, we encourage authorship decisions to favor inclusivity and equity.
:::
```
